### ISSUE TITLE
Question/Clarification:  How to open any web link on host?

### ISSUE THREAD

**Issue by robotictacos** [casualsnek/cassowary#89]
The documentation clearly states that in order to open a file type on the host, the hstopen batch file needs to be set as the default app.  This definitely seems to work in some cases.  My particular use case is I am running Outlook 365 in the VM.  If I click on a PDF file, that will open on my host because I've set the batch file as the default app for PDF files.  However, I would like to be able to click on web links contained within emails and have those open in my host browser, but so far I don't see a way to set hstopen as the default web browser.  I'm probably missing something stupid that I should have read, can you point me in the right direction?

**Comment by jamespharvey20** [comment]
> hstopen

For others not familiar with this, hostopen :-)

**Comment by jamespharvey20** [comment]
Which version of Windows?

I gave this a few minutes, and couldn't find a way to set `hostopen.bat` as the default browser on Windows 10.  The typical default apps interface doesn't seem to give me a way to select programs not in the list.  I was able to set it for locally saved `.html` files, but I know that's not what you're asking for.  I made a dummy `.html` file.  I right clicked it, cliekd "Open With", "Choose another app", "More apps", clicked "Always use this app to open .html files", "Look for another app on this PC", and typed "C:\Program Files\Cassowary\hostopen.bat".

I also tried running `ftype http="C:\Program Files\cassowary\hostopen.bat" %1` but that didn't affect the default browser.

**Comment by robotictacos** [comment]
Yes, Windows 10.    I had the same experience, I can set htm/html locally saved files to open on the host, but I couldn't find a way to allow me to click a link and open said link on the host.
  
  
  
  
  
  
  

  
  
>

**Comment by Bill5122** [comment]
For anyone still trying to do this, i followed the stackoverflow answer below, using hostopen.bat as the executable.

https://superuser.com/questions/1750214/win-11-regedit-set-default-app-for-http-https-launch-from-app-e-g-npm-run
